### PR TITLE
Override default HTTP User-Agent and make it tuneable

### PR DESCRIPTION
**Pull request by smortex**
The default User-Agent for Net::HTTP is "Ruby".

image

In order to make it easier to distinguish traffic from various sources, change this default User-Agent string with a custom one based on the riemann tool name and version, and also include the project URL in the same way bots report their URL.

### CODE REVIEW

File: lib/riemann/tools/apache_status.rb [hunk 1]
```diff
@@ -14,9 +14,10 @@ class ApacheStatus
       require 'uri'
 
       opt :uri, 'Apache Server Status URI', default: 'http://localhost/server-status'
+      opt :user_agent, 'User-Agent header to use for HTTP requests', short: :none, default: "#{File.basename($PROGRAM_NAME)}/#{Riemann::Tools::VERSION} (+https://github.com/riemann/riemann-tools)"
```
Comment (jamtur01) [inline lib/riemann/tools/apache_status.rb:17]: Minor nit. I'd chop the "to use" out of this - shorten it.

Comment (jamtur01) [reply lib/riemann/tools/apache_status.rb:17]: Same for all other uses, etc.

**Review by jamtur01 — DISMISSED** [review]
LGTM. Minor nit.

File: lib/riemann/tools/apache_status.rb (hunk 1 above)
Comment (smortex) [reply lib/riemann/tools/apache_status.rb]: Did that and added the parameter that slept through for consul.

**Review by jamtur01 — APPROVED** [review]
LGTM

**Pull request merged**